src/wcm.hpp

```diff
@@ -39,6 +41,8 @@
 
 #include "metadata.hpp"
 
+#define _(MSG) gettext(MSG)
```

Comment:
Maybe `#include <glibmm/i18n.h>` instead of manually defining the macro?

Reply:
OK